projects/storefrontlib/cms-components/content/paragraph/paragraph.component.spec.ts

```diff
@@ -46,6 +60,10 @@ describe('CmsParagraphComponent in CmsLib', () => {
             provide: CmsComponentData,
             useValue: MockCmsComponentData,
           },
+          {
+            provide: FeatureConfigService,
+            useClass: MockFeatureConfigService,
```

Comment:
`should:` Because `FeatureConfigService` is optional, I believe it's worth adding unit tests that cover such a scenario.

Reply:
Seems a bit odd  to test since the behaviour should be the same or we would test that a bug occurs in case the feature config service is not imported. This mock service will also be removed in 6.0 after cleanup as a result of the earlier cleanup TODO.

Reply:
We could check whether `navigate` has been called instead of `navigateByUrl` but yes, such a test will have to be removed for 6.0. Adding this is totally up to you 🙂